### PR TITLE
Add hero image and parallax

### PR DESCRIPTION
**Pull request by dellisd**
Still needs a proper colour for the hero text.

### CODE REVIEW

File: src/pages/home.tsx [hunk 1]
```diff
@@ -11,21 +22,94 @@ const HeroSection = styled.div`
...
+    <HeroLayerWrapper speed={-1.5 * (layerImages.length - 1 - index)}>
+      <HeroLayer src={image} />
+    </HeroLayerWrapper>
+  ));
+
+  const textLayer = 5;
+  const front = layers.slice(0, textLayer);
+  const back = layers.slice(textLayer);
+  const text = (
+    <HeroLayerWrapper speed={-1.5 * (layerImages.length - 1 - textLayer)}>
```
Comment (Walsker) [inline src/pages/home.tsx:79]:
```suggestion
    <HeroLayerWrapper
       key='text-0'
       speed={-1.5 * (layerImages.length - 1 - textLayer)}
    >
```

**Review by Walsker — CHANGES_REQUESTED** [review]

**Comment by Walsker** [comment]
Also, since the "text" layer is wedged between the image layers in the parallax, how can we make it so a button is clickable once we put one there?

**Comment by dellisd** [comment]
> Also, since the "text" layer is wedged between the image layers in the parallax, how can we make it so a button is clickable once we put one there?

I added `pointer-events: none` to the layers and `pointer-events: all` to the text (which would include the button).

**Review by Walsker — APPROVED** [review]
Looks good!

**Pull request merged**